crowbar_framework/app/assets/javascripts/application.js

```diff
@@ -105,6 +104,7 @@ jQuery(document).ready(function($) {
 
   setBlockUI('submit', $('[data-blockui]'));
   setBlockUI('click', $('body[class="installer/upgrades"] [data-blockui]'));
+  setBlockUI('click', $('body.nodes').find('[data-blockui]'));
```

Comment:
Instead of adding more and more `setBlockUI` calls you should just split it up into 2 data attributes, e.g.

```
  $('[data-blockui]').live('submit', function(event) {
    $.blockUI({
      css: {
        border: 'none',
        padding: '15px',
        backgroundColor: '#000',
        '-webkit-border-radius': '10px',
        '-moz-border-radius': '10px',
        opacity: .5,
        color: '#fff'
      },
      message: $(event.target).data('blockui')
    });
  });

  $('[data-blockui-click]').live('click', function(event) {
    $.blockUI({
      css: {
        border: 'none',
        padding: '15px',
        backgroundColor: '#000',
        '-webkit-border-radius': '10px',
        '-moz-border-radius': '10px',
        opacity: .5,
        color: '#fff'
      },
      message: $(event.target).data('blockui-click')
    });
  });
```